### PR TITLE
Add issue templates for bugs, stories, and tasks

### LINKED ISSUE #4: Draft Issue Templates
Having issue templates will make the creation of different issues faster and more efficient. It also prevents missing sections and details needed to describe an issue.

### PR DESCRIPTION
**Pull request by luigibesani**
Add issue templates for bug reports, user stories, and tasks.
Closes #4

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/task.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+## Description
+<!-- Provide a general summary of the feature in the title above -->
+
+## Possible Implementation
+<!-- If you have an idea of how to implement the issue, jot it down here - this section is optional -->
+
+## Out of scope
+<!-- The portion of this story that does not fall under the responsibility of the application -->
+
```
Comment (pascwhale) [inline .github/ISSUE_TEMPLATE/task.md:18]: We don't need an empty line here.

File: .github/ISSUE_TEMPLATE/user-story.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+<!-- Descirbe the actions the main stakeholder should be able to take -->
+<!-- eg: As a user with a CSU Office 365 account, I want to be able to use it to login. -->
+
+## Narrative
+<!-- The typical flow of the story on the application -->
+<!-- eg: As an administrator of the system it should be possible for me to add my Microsoft organization credentials to the system in order to allow SSO functionality for users in said organization. -->
+
+## Out of scope
+<!-- The portion of this story that does not fall under the responsibility of the application -->
+
```
Comment (pascwhale) [inline .github/ISSUE_TEMPLATE/user-story.md:20]: We don't need an empty line here.

**Review by pascwhale — CHANGES_REQUESTED** [review]
Can we remove empty lines between a section hint and its example?

**Review by m-triassi — APPROVED** [review]
I'll let you decide if you want to address @pascwhale's comments or not, for me this seems like a good template following our initial drafts of these from our conversations.

Once you deal with Pascal's comments i'd say you can just go ahead and merge

**Review by pascwhale — APPROVED** [review]
Great, thank you for removing them.

**Pull request merged**